hexa/workspaces/admin.py

```diff
@@ -42,3 +48,16 @@ class ConnectionFieldAdmin(admin.ModelAdmin):
         "connection",
         "code",
     )
+
+
+@admin.register(WorkspaceInvitation)
+class WorkspaceInvitationAdmin(admin.ModelAdmin):
+    list_display = (
+        "id",
```

Comment:
We can probably skip the `id` and keep `email` as the first column

Reply:
Okay.